tests/target/issue-3669.rs

```diff
@@ -0,0 +1,51 @@
+pub trait PCG:
```

Comment:
Could you add a test to make sure that the comment after `+` won't get removed as well? E.g.,

```rust
    self::sealed::Sealed + // comment1
      Sized +                // comment2
      Eq +                   // comment3
      Hash +                // comment4
      Debug +               // comment5
      Clone +               // comment6
      Default +             // comment7
      Serialize +           // comment8
      for<'a> Deserialize<'a> // comment9
```

Reply:
See commit https://github.com/rust-lang/rustfmt/pull/3680/commits/54da77abc2e9137f3ef4356d4af56545b87789cf for the test.
Thanks to that, I fixed another issue where post comments were not aligned because they had a trailing newline: there is a test for that to prevent alignment in `ListItem::is_different_group`.